cmd/browse.go

```diff
@@ -24,8 +25,13 @@ func init() {
 
 func runBrowseServer(cmd *cobra.Command, args []string) {
 
+	vm, err := providerServices.NewJSONFileProviderVersionManager()
+	if err != nil {
+		panic(err)
+	}
+
 	restAPIServer := browse.New(registrar.NewRegistrarGrpcClient(registrar.RegistrarServiceEndpoint),
 		version_manager.NewVersionManagerGrpcClient(version_manager.VersionManagerEndpoint),
-		release.NewBrowseGrpcClient(release.ReleaseServiceEndpoint))
+		release.NewBrowseGrpcClient(release.ReleaseServiceEndpoint), vm)
```

Comment:
could you use a better name for vm?

Reply:
Yes, updated it with version_manager_svc